### PR TITLE
Update item.lua

### PR DESCRIPTION
**Pull request by PaladinFallen**

### CODE REVIEW

File: prototypes/item/item.lua [hunk 1]
```diff
@@ -6,10 +6,10 @@ data:extend(
 		icon = "__Exoskeleton Mark II__/graphics/exoskeleton-mark-ii.png",
 		icon_size = 32,
 		placed_as_equipment_result = "exoskeleton-mark-ii",
-		flags = {"goes-to-main-inventory"},
+		--flags = { "goes-to-quickbar" },
```
Comment (axtox) [inline prototypes/item/item.lua]: you can safely remove this code line, i've tested it - works fine without it

**Review by axtox — CHANGES_REQUESTED** [review]
Make prior changes to merge this pull request! See comments in "files" section

**Comment by PaladinFallen** [comment]
I made the fixes to the files, so everything is good.

**Review by axtox — APPROVED** [review]
Okay, now it can be merged! Thanks!

**Pull request merged**